src/test/binaryutils_test.cpp

```diff
@@ -24,11 +24,29 @@ using namespace ovms;
 
 namespace {
 
-class BinaryUtilsTest : public ::testing::Test {};
+class BinaryUtilsTest : public ::testing::Test {
+protected:
+    void SetUp() override {
+        std::ifstream DataFile;
```

Comment:
Repeated code, this could be utility function and could be used in both places - here and in RestParserBinaryInputs.